baynet/parameters.py

```diff
@@ -52,6 +52,8 @@ def dfe_estimate(
     ) -> None:
         """Predict parameters using DFE method."""
         self.rescale_probabilities()
+        if iterations < 1:
```

Comment:
should `iterations` be optional, and this be a check if it's `None`? rn it's gonna default to 250 and skip this